Sofa/GUI/Qt/src/sofa/gui/qt/RealGUI.cpp

```diff
@@ -381,7 +381,7 @@ RealGUI::RealGUI ( const char* viewername)
     connect ( timerStep, SIGNAL ( timeout() ), this, SLOT ( step() ) );
     connect ( this, SIGNAL ( quit() ), this, SLOT ( fileExit() ) );
     connect ( startButton, SIGNAL ( toggled ( bool ) ), this , SLOT ( playpauseGUI ( bool ) ) );
-    connect ( ResetSceneButton, SIGNAL ( clicked() ), this, SLOT ( resetScene() ) );
+    connect ( ReloadSceneButton, SIGNAL ( clicked() ), this, SLOT ( fileReload() ) );
```

Comment:
It would make more sens to add another button reload side by side with reset one.
Does it means you want to progressively remove the reset option?